TodoReview.py

```diff
@@ -150,6 +150,8 @@ def run(self, edit, **args):
 		window = self.view.window()
 		paths = args.get('paths', None)
 		settings = Settings(self.view, args.get('settings', False))
+		if not settings.get('current_file', False):
+			paths = [self.view.file_name()]
```

Comment:
Yeah, that is actually a good point. I hadn't considered that scenario. I agree completely. Thanks for the review @FichteFoll